### ISSUE TITLE
filterscripts not working due to outdated phys_services.spe_fit_injector (Trac #2230)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2222]
Hello,

(documenting discussion from slack so this does not get lost)
trying to run filterscripts IceCube_BaseProc.py but I see this error in the current trunk:

```text
File "/mnt/lfs3/user/flauber/software/py2-v3.0.1/combo_trunk/build/lib/icecube/phys_services/spe_fit_injector.py", line 90, in Calibration
    spe_charge_dist.exp_amp = self.fit_dict[omkey]['exp_amp']
AttributeError: *** The dynamism of this class has been disabled
*** Attribute (exp_amp) does not exist in class SPEChargeDistribution
```

This is apparently due to an unupdated spe_fit_injector in phys-services what ever this means.

As this is in filterscripts and the TFT proposal season already in full swing, I would like to raise the priority of this by one (this is under the assumption that the trunk will be used for the upcoming  session and not for the season after that)

Cheers,
Frederik

Migrated from https://code.icecube.wisc.edu/ticket/2230
```json
{
    "status": "closed", 
    "changetime": "2019-03-13T20:47:56", 
    "description": "Hello,\n\n(documenting discussion from slack so this does not get lost)\ntrying to run filterscripts IceCube_BaseProc.py but I see this error in the current trunk:\n\n{{{\nFile \"/mnt/lfs3/user/flauber/software/py2-v3.0.1/combo_trunk/build/lib/icecube/phys_services/spe_fit_injector.py\", line 90, in Calibration\n    spe_charge_dist.exp_amp = self.fit_dict[omkey]['exp_amp']\nAttributeError: *** The dynamism of this class has been disabled\n*** Attribute (exp_amp) does not exist in class SPEChargeDistribution\n}}}\n\nThis is apparently due to an unupdated spe_fit_injector in phys-services what ever this means.\n\nAs this is in filterscripts and the TFT proposal season already in full swing, I would like to raise the priority of this by one (this is under the assumption that the trunk will be used for the upcoming  session and not for the season after that)\n\nCheers,\nFrederik\n\n\n", 
    "reporter": "flauber", 
    "cc": "saxani", 
    "resolution": "fixed", 
    "_ts": "1552510076492885", 
    "component": "infrastructure", 
    "summary": "filterscripts not working due to outdated phys_services.spe_fit_injector", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2019-01-15T09:43:47", 
    "milestone": "Vernal Equinox 2019", 
    "owner": "olivas", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20190115T14:35:30: 
* olivas changed status from "new" to "accepted"
* olivas changed owner from "" to "olivas"
* olivas changed milestone from "" to "Vernal Equinox 2019"
* olivas changed priority from "major" to "blocker"

**Comment by icecube-trac** [comment]
Trac update at 20190116T14:13:44: david.schultz commented:

> I guess the correct one is in sim-services.  Could these be merged, so there is one true way to do this?
> 
> I also second raising the priority, so we can get started with testing pass3 processing with a mostly frozen set of scripts.

**Comment by icecube-trac** [comment]
Trac update at 20190227T01:26:12: 
* olivas commented:

> https://code.icecube.wisc.edu/projects/icecube/changeset/171590/IceCube/meta-projects/combo/trunk/phys-services
> 
> There is only one SPEFitInjector now that understands both old and new JSON schemas.
> 
> Making progress.  It should run without crashing...at first.  No guarantees about the quality of results.  The mean charges for ATWD and FADC are still not set, but Spencer is going to take a look and push this over the goal-line.
> 


* olivas changed cc from "" to "saxani"

**Comment by icecube-trac** [comment]
Trac update at 20190313T20:47:56: 
* olivas changed resolution from "" to "fixed"
* olivas commented:

> In changeset 171964:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="171964"
> works again for both file types.  closes #2222.
> ```


* olivas changed status from "accepted" to "closed"
